### PR TITLE
Add annotations.proto and generated Go code

### PR DESCRIPTION
**Pull request by pkwarren**
In order to prepare a PR for
https://github.com/google/gnostic/issues/397, gnostic-models needs to be updated to include openapiv3/annotations.proto (and its generated code).

### CODE REVIEW

File: openapiv3/annotations.pb.go [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.27.1
+// 	protoc        v3.19.3
```
Comment (pkwarren) [inline openapiv3/annotations.pb.go:18]: Generated this with the same tool versions used to generate openapiv3/OpenAPIv3.proto code.

File: openapiv3/annotations.proto [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+syntax = "proto3";
+
+package openapi.v3;
+
+import "google/protobuf/descriptor.proto";
+import "openapiv3/OpenAPIv3.proto";
+
+// The Go package name.
+option go_package = "./openapiv3;openapi_v3";
```
Comment (pkwarren) [inline openapiv3/annotations.proto:23]: This is the only change to this file (matching the one made to OpenAPIv3.proto).

**Comment by pkwarren** [comment]
@timburks - Are you the right person to tag for reviews on these PRs? I was hoping to get some feedback on this and the downstream fix in https://github.com/google/gnostic/pull/400 but happy to ping another reviewer.

**Comment by timburks** [comment]
@pkwarren Thanks, AFAIK this is only used by @jefftree and the k8s team, so since they're ok with https://github.com/google/gnostic/issues/397, I think it's fine to proceed with this. Merging...

**Pull request merged**